src/Common/src/System/Security/Cryptography/Asn1/asn.xsd

```diff
@@ -31,7 +31,7 @@
         <xs:element name="BitString" type="FieldBase" />
         <xs:element name="NamedBitList" type="EnumerationType" />
         <xs:element name="OctetString" type="FieldBase" />
-        <xs:element name="ObjectIdentifier" type="FieldBase" />
+        <xs:element name="ObjectIdentifier" type="ObjectIdentifierType" />
```

Comment:
An ObjectIdentifierCollectionType should also be defined, and the collection version changed to use it.

Reply:
Oops, I thought I did that, but apparently I forgot to.